app/models/manageiq/providers/openstack/cloud_manager/vm/operations.rb

```diff
@@ -8,7 +8,14 @@ module ManageIQ::Providers::Openstack::CloudManager::Vm::Operations
 
   def raw_destroy
     raise "VM has no #{ui_lookup(:table => "ext_management_systems")}, unable to destroy VM" unless ext_management_system
-    with_provider_object(&:destroy)
+    with_notification(:vm_destroy,
+                      :options => {
```

Comment:
Thanks for the PR, the approach looks good to me.

I think, ```:instance_name => ...``` should not be wrapped in ```:options => ...```

Reply:
In fact it should - with_notification uses keyword parameter `:options` not the ruby "options hash".

Reply:
Ah, thanks, I forgot about this Ruby 2.0 update.